homeassistant/components/sfr_box/quality_scale.yaml

```diff
@@ -0,0 +1,96 @@
+rules:
```

Comment:
Why can the coordinator state be None? As in, I understand that it can be None if it doesn't support it, but wouldn't these coordinators not be created at all?